src/challenge.js

```diff
@@ -210,10 +217,9 @@ module.exports = (fcrToken, LMSR, web3, id, address, defaultOptions) => {
 
     const decision = decisionForOutcome(outcome)
     const decisionMarket = await getDecisionMarket(decision)
-    const outcomeCost = await calculateOutcomeCost(outcome, amount)
-    const outcomeFee = await calculateOutcomeFee(outcome, amount)
+    const outcomeCost = parseInt(await calculateOutcomeCost(outcome, amount))
```

Comment:
do we need these? they might fail if the number is too big

Reply:
`totalOutcomeCost` was adding two strings together. 2 + 0 = 20.

Reply:
ah ok good catch!!

Reply:
better to leave this than mess around with BigNumber .. although technically this will need to be done with BigNumber in order for this to work with ... big numbers